File: app/src/main/java/com/niuda/a3jidi/laok/booklook/model/IBookView.java
package com.niuda.a3jidi.laok.booklook.model;

import com.niuda.a3jidi.lib_base.base.base.IBaseView;

/**
 * (￣▽￣)"
 *
 * @version V1.0 :
 * @author: Administrator
 * @date: 2017-11-12 15:40
 */

public interface IBookView extends IBaseView {
    void onSuccess(Book book);
    void onError(String result);
}

File: app/src/main/java/com/niuda/a3jidi/laok/booklook/model/IBookView.kt
package com.niuda.a3jidi.laok.booklook.model

import com.niuda.a3jidi.lib_base.base.base.IBaseView

/**
 * (￣▽￣)"
 *
 * @version V1.0 :
 * @author: Administrator
 * @date: 2017-11-12 15:40
 */

interface IBookView : IBaseView {
    fun onSuccess(book: Book)
    fun onError(result: String)
}
